### ISSUE TITLE
Websocket as dependency

### ISSUE THREAD

**Issue by ZoranPandovski** [aurelion314/pysurrealdb#2]
Since version 0.3 uses WebSockets by default, do we need to include the `websocket` as a required dependency?

**Comment by aurelion314** [comment]
Good point. In the latest release, websocket is not actually default because I didn't feel comfortable with the level of testing I had done on it. I updated the readme to clarify that you'll need to install websockets if you want to use it.

If websockets gets more use and I return that to the default, I will be sure to add it to the requirements.

Thanks!
